### DISCUSSION TITLE
Cannot find module after install devDependency

### DISCUSSION THREAD

**Discussion by yuniit** [faker-js/faker#1135] [Q&A] [answered]
I just install this library in my `meteor` project follow the document `install @faker-js/faker --save-dev`

After import 
```js
import { faker } from '@faker-js/faker';
```
my app crash and display the following error: 
```js
Error: Cannot find module "/node_modules/@faker-js/faker/dist/cjs/index.js". Try installing the npm package or make sure it is not a devDependency.
```
Should I install Dependency instead?

**Comment by Shinigami92** [comment, marked as answer]
Yes, seems you want to use Faker not for e.g. mocking test data or so
So if you want to use Faker in production bundled code, sure you need to install it as dependency directly

You should think about using it like so:
```ts
import { faker } from '@faker-js/faker/locale/en';
```

Because this will reduce bundle size from ~4 MiB to ~600 KiB
